src/engine/enginevumeter.cpp

```diff
@@ -85,14 +91,25 @@ void EngineVuMeter::process(CSAMPLE* pIn, const int iBufferSize) {
         m_fRMSvolumeSumR = 0;
     }
 
-    if (clipped) {
-        m_ctrlPeakIndicator->set(1.);
-        m_peakDuration = PEAK_DURATION * sampleRate / iBufferSize / 2000;
-    } else if (m_peakDuration <= 0) {
-        m_ctrlPeakIndicator->set(0.);
+    if (clipped == SampleUtil::CLIPPING_LEFT || clipped == SampleUtil::CLIPPING_BOTH) {
```

Comment:
this should be 
```
if (clipped & SampleUtil::CLIPPING_LEFT) {
```
Or if one does not like the & operator hidden in helper function isClippingLeftSet()